xml/issue4425.xml

```diff
@@ -19,6 +19,11 @@ member. As accessing a data member can never throw exception and <tt>function_re
 support <tt>noexcept</tt> qualifed function types, the deduction guide should be 
 updated to produce <tt>noexcept</tt>-qualified signature.</p>
 
+
+<note>2025-10-21; Reflector poll.</note>
+<p>
+Set status to Tentatively Ready after six votes in favour during reflector poll.
```

Comment:
I usually leave the poll long than one day to see if anybody objects, but I think this one is not going to be controversial so let's change it.

Reply:
I was also wondering about it, but it seemed uncontroversial, so went ahead.